docs/DeveloperGuide.md

```diff
@@ -1,34 +1,965 @@
+
+The usage scenario of remove from watchlist is similar to the add to watchlist command, but we will be using `RemoveCommand`, `RemoveCommandParser` and `Watchlist#removeAnimeFromList()`.
+
+> :bulb: ANIME_ID_IN_WATCHLIST is the index of anime inside the watchlist itself, not the anime index in AnimeData.
+
+#### 4.7.2 Design Consideration
+This section describes the design considerations taken when implementing this feature.
+
+Aspect: **which index to use when removing an anime**
+
+| Approach | Pros | Cons  |
+| --- | --- | --- |
+| 1. Use anime ID itself | Users who remember the anime ID will be able to remove the anime quicker | The program will have to search through the entire list of anime in the watchlist for that particular anime ID  |
+| 2. Use index of anime in the watchlist | Faster for the program to delete a particular index | User will have to view the watchlist to check which index the anime is at |
+
+We have decided to use approach 2 instead of 1, as it will not only will it be much faster for the program to delete one particular index instead of having to search through the whole list, it would be provide better user experience as the user will not need to search for the anime ID that he wants to remove in the case that he does not know the anime ID.
+
+<br/>
+
+### 4.8 View All Anime in Watchlist
+Users can view the anime that they have stored in a specific watchlist by using the `view` command. In doing so, they can easily check what anime they would like to watch next.
+
+#### 4.8.1 Current Implementation
+The `view` command is currently implemented by the `ViewWatchlistCommand`. It is instantiated by `Parser` and executed by `Main`. This allows users to view all the anime currently inside a `Watchlist` object. 
+
+An example usage scenario on how view anime in watchlist behaves is given below.
+
+**Step 1:** In the `Main` class, the user will input `view -v <WATCHLIST_ID>`. `Ui` will take in this input and is passed into `Parser`.
+
+**Step 2:** In `Parser’, `view` will be extracted out of the input, leading to a new `ViewWatchlistParser` object being instantiated, and in the constructor, a new `ViewWatchlistCommand` is created.
+
+**Step 3:** `ViewWatchListParser#parse()` is then called in `Parser`, which will validate the parameter that was given by the user. If the parameter is correct, the watchlist index will be set in the `ViewWatchlistCommand` object. The `ViewWatchlistCommand` object is then returned back to `Parser`, and back to `Main`.
+
+**Step 4:** The `ViewWatchlistCommand#execute()` would then be called by `Main`, in which the WATCHLIST_ID will be validated, and if valid, a string containing all the anime name inside the active `Watchlist` will be built and returned to `Main`, where it will be printed out by `Ui`.
+
+**Step 5:** `ViewWatchlistParser`, `ViewWatchlistCommand`, `Parser` and `Command` are terminated
+
```

Comment:
I suggest maybe having a diagram for certain steps here like for your other implementations for better understanding